# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1654800471 -7200
# Parent  fd7ebcf0506e8d818a58f21a96b59449afcc6e65
Misc: Suppress console message about Almost Standards and Quirks mode.

diff --git a/parser/html/nsHtml5DocumentBuilder.cpp b/parser/html/nsHtml5DocumentBuilder.cpp
--- a/parser/html/nsHtml5DocumentBuilder.cpp
+++ b/parser/html/nsHtml5DocumentBuilder.cpp
@@ -76,21 +76,25 @@ void nsHtml5DocumentBuilder::SetDocument
   const char* errMsgId = nullptr;
 
   switch (m) {
     case STANDARDS_MODE:
       mode = eCompatibility_FullStandards;
       break;
     case ALMOST_STANDARDS_MODE:
       mode = eCompatibility_AlmostStandards;
+#ifndef MOZ_THUNDERBIRD
       errMsgId = "errAlmostStandardsDoctypeVerbose";
+#endif
       break;
     case QUIRKS_MODE:
       mode = eCompatibility_NavQuirks;
+#ifndef MOZ_THUNDERBIRD
       errMsgId = "errQuirkyDoctypeVerbose";
+#endif
       break;
   }
   mDocument->SetCompatibilityMode(mode);
   if (errMsgId) {
     nsContentUtils::ReportToConsole(
         nsIScriptError::warningFlag, "HTML_PARSER__DOCTYPE"_ns, mDocument,
         nsContentUtils::eHTMLPARSER_PROPERTIES, errMsgId);
   }
